prov/efa/src/rxr/rxr_msg.c

```diff
@@ -1047,15 +1047,14 @@ ssize_t rxr_msg_generic_recv(struct fid_ep *ep, const struct fi_msg *msg,
 		goto out;
 	}
```

Comment:
Couple typos in the commit message for ```prov/efa: change the way pkt_entry matching with rx_entry in zero copy
```

communcations -> communications

laster remove -> later removed

Reply:
Thank you! fixed